### DISCUSSION TITLE
Migrate Kafka clusters from VM to OpenShift Via MirrorMaker2

### DISCUSSION THREAD

**Discussion by AvihuHenya** [strimzi/strimzi-kafka-operator#6128] [Q&A]
I am trying to set up a POC to prove we can use strimzi to migrate clusters deployed in VMs to my Strimzi environment running in OpenShift.  
Is there any problems with this approach?
Is there any additional configuration/information I need to check before I start working?

**Comment by scholzj** [comment]
Well, you can use for example Mirror Maker 2 for it. So I guess you should check the MM2 docs etc.
